src/stampit.js

```diff
@@ -128,12 +131,9 @@ function extractStatics(targ, src) {
  */
 function compose(...factories) {
   let stamps = factories.slice(),
-      result = stampit(),
-      state = { state: {} },
-      methods = {},
-      statics = {};
-
-  result.compose = compose;
+      result = stampit.init(),
```

Comment:
Interesting... the `.init()` shouldn't be required here, but it kindof describes what you're doing and should result in a new stamp as expected...